style.css

```diff
@@ -17,7 +17,7 @@ body {
     padding: 20px;
     box-shadow: #00000011 0px 0px 15px 5px;
     width: 500px;
-    min-width: fit-content;
+    min-width:500px;
```

Comment:
just why?:))

Reply:
because "This is an educational project" XD